[2023-01-26T21:47:13.316+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: better_workflow_3.transition_best_model_version_to_prod_precision_task manual__2023-01-26T21:42:31.315906+00:00 [queued]>
[2023-01-26T21:47:13.329+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: better_workflow_3.transition_best_model_version_to_prod_precision_task manual__2023-01-26T21:42:31.315906+00:00 [queued]>
[2023-01-26T21:47:13.329+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-26T21:47:13.329+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 4
[2023-01-26T21:47:13.330+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-26T21:47:13.349+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): transition_best_model_version_to_prod_precision_task> on 2023-01-26 21:42:31.315906+00:00
[2023-01-26T21:47:13.368+0000] {standard_task_runner.py:55} INFO - Started process 49453 to run task
[2023-01-26T21:47:13.389+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'better_workflow_3', 'transition_best_model_version_to_prod_precision_task', 'manual__2023-01-26T21:42:31.315906+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_dag.py', '--cfg-path', '/tmp/tmp3ww87ypi']
[2023-01-26T21:47:13.394+0000] {standard_task_runner.py:83} INFO - Job 21: Subtask transition_best_model_version_to_prod_precision_task
[2023-01-26T21:47:13.423+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2023-01-26T21:47:13.492+0000] {task_command.py:388} INFO - Running <TaskInstance: better_workflow_3.transition_best_model_version_to_prod_precision_task manual__2023-01-26T21:42:31.315906+00:00 [running]> on host 69914ba18a5b
[2023-01-26T21:47:13.598+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=baahmedabdessamad@gmail.com
AIRFLOW_CTX_DAG_OWNER=abdessamad
AIRFLOW_CTX_DAG_ID=better_workflow_3
AIRFLOW_CTX_TASK_ID=transition_best_model_version_to_prod_precision_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-26T21:42:31.315906+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-26T21:42:31.315906+00:00
[2023-01-26T21:47:13.763+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:13 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh()

All git commands will error until this is rectified.

This initial warning can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|none|n|0: for no warning or exception
    - warn|w|warning|1: for a printed warning
    - error|e|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2023-01-26T21:47:14.394+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:14 WARNING mlflow.pyfunc: Detected one or more mismatches between the model's dependencies and the current Python environment:
 - pandas (current: 1.5.2, required: pandas==1.4.2)
 - scikit-learn (current: 1.2.1, required: scikit-learn==1.1.0)
 - xgboost (current: 1.7.3, required: xgboost==1.7.2)
To fix the mismatches, call `mlflow.pyfunc.get_model_dependencies(model_uri)` to fetch the model's environment and install dependencies using the resulting environment file.
[2023-01-26T21:47:14.787+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:14 WARNING mlflow.pyfunc: Detected one or more mismatches between the model's dependencies and the current Python environment:
 - pandas (current: 1.5.2, required: pandas==1.4.2)
 - scikit-learn (current: 1.2.1, required: scikit-learn==1.1.0)
 - xgboost (current: 1.7.3, required: xgboost==1.7.2)
To fix the mismatches, call `mlflow.pyfunc.get_model_dependencies(model_uri)` to fetch the model's environment and install dependencies using the resulting environment file.
[2023-01-26T21:47:15.306+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:15 WARNING mlflow.pyfunc: Detected one or more mismatches between the model's dependencies and the current Python environment:
 - pandas (current: 1.5.2, required: pandas==1.4.2)
 - scikit-learn (current: 1.2.1, required: scikit-learn==1.1.0)
 - xgboost (current: 1.7.3, required: xgboost==1.7.2)
To fix the mismatches, call `mlflow.pyfunc.get_model_dependencies(model_uri)` to fetch the model's environment and install dependencies using the resulting environment file.
[2023-01-26T21:47:15.780+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:15 WARNING mlflow.pyfunc: Detected one or more mismatches between the model's dependencies and the current Python environment:
 - pandas (current: 1.5.2, required: pandas==1.4.2)
 - psutil (current: 5.9.4, required: psutil==5.9.0)
 - scikit-learn (current: 1.2.1, required: scikit-learn==1.1.0)
 - typing-extensions (current: 4.4.0, required: typing-extensions==4.2.0)
 - xgboost (current: 1.7.3, required: xgboost==1.7.2)
To fix the mismatches, call `mlflow.pyfunc.get_model_dependencies(model_uri)` to fetch the model's environment and install dependencies using the resulting environment file.
[2023-01-26T21:47:16.329+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:16 WARNING mlflow.pyfunc: Detected one or more mismatches between the model's dependencies and the current Python environment:
 - pandas (current: 1.5.2, required: pandas==1.4.2)
 - psutil (current: 5.9.4, required: psutil==5.9.0)
 - scikit-learn (current: 1.2.1, required: scikit-learn==1.1.0)
 - typing-extensions (current: 4.4.0, required: typing-extensions==4.2.0)
 - xgboost (current: 1.7.3, required: xgboost==1.7.2)
To fix the mismatches, call `mlflow.pyfunc.get_model_dependencies(model_uri)` to fetch the model's environment and install dependencies using the resulting environment file.
[2023-01-26T21:47:17.248+0000] {logging_mixin.py:137} WARNING - 2023/01/26 21:47:17 WARNING mlflow.pyfunc: Detected one or more mismatches between the model's dependencies and the current Python environment:
 - pandas (current: 1.5.2, required: pandas==1.4.2)
 - psutil (current: 5.9.4, required: psutil==5.9.0)
 - scikit-learn (current: 1.2.1, required: scikit-learn==1.1.0)
 - typing-extensions (current: 4.4.0, required: typing-extensions==4.2.0)
 - xgboost (current: 1.7.3, required: xgboost==1.7.2)
To fix the mismatches, call `mlflow.pyfunc.get_model_dependencies(model_uri)` to fetch the model's environment and install dependencies using the resulting environment file.
[2023-01-26T21:47:19.995+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-26T21:47:20.007+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=better_workflow_3, task_id=transition_best_model_version_to_prod_precision_task, execution_date=20230126T214231, start_date=20230126T214713, end_date=20230126T214720
[2023-01-26T21:47:20.035+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-26T21:47:20.050+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
